amida-tech/hl7-telescoper-api-server
# Hl7 19 am get file endpoint route 2

### Related JIRA tickets:
- https://jira.amida.com/browse/HL7-19

### What exactly does this PR do?
- Expands upon HL7-6 ticket that creates GET endpoint to retrieve file; except this time, 
a whole Object is returned rather than just the raw HL7 document
- The object contains the filename, the dateAdded ( this is just a placeholder for an actual user ID),
and an string array of messages
- The messages array will be replaced by an object array of messages that contain the raw message along with its parsed version when the parse schema is merged. For now, there is just placeholder text to indicate where the parsed version will go.


### Manual test cases?
- You can use the 500HL7Messages.txt-1561496513175 file but only keep two or three for readability's sake
- Enter that into a GET request in Postman (e.g http://localhost:4040/api/hl7/500HL7Messages.txt-1561496513175)
- You should see the filename, dateAdded, and a messages array
